src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -285,10 +285,10 @@ private static CommandLineProgram extractCommandLineProgram(final String[] args,
         classes.addAll(simpleNameToClass.values());
 
         if (args.length < 1) {
-            printUsage(classes, commandLineName);
+            printUsage(System.err, classes, commandLineName);
```

Comment:
Does this case actually represent an error?

Reply:
lets move this into the normal help output case, so we don't have a division between them.

Reply:
Done. So now it goes to stdout.